modules/openapi-generator/src/main/resources/dart2/enum_inline.mustache

```diff
@@ -1,14 +1,14 @@
 {{#description}}/// {{{description}}}{{/description}}
-class {{{classname}}}{{{enumName}}} {
+class {{{enumName}}} {
```

Comment:
I think that's a big breaking change for no big benefit, enum serialisation still won't be as good as in other languages. I think it's time to give up on doing this in openapi and use a more popular 3rd party library do it for us. I'm only familiar with json_serializable but it doesn't handle standalone enums so can't use it here. I'll look around soon.

Reply:
This is actually part of  #8220 but required for this to work.